profiler/profiler_test.go

```diff
@@ -241,58 +243,6 @@ func TestStopLatency(t *testing.T) {
 	}
 }
 
-func TestProfilerInternal(t *testing.T) {
```

Comment:
Note: We could "fix" this test by removing the `tick <- time.Now()` line and changing `p.exit <- struct{}{}` to `close(p.exit)`. That would make the test pass. However I personally think this test should just be removed.